packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart

```diff
@@ -4,14 +4,20 @@
 
 part of google_maps_flutter;
 
+final GoogleMapsFlutterPlatform _googleMapsFlutterPlatform =
+    GoogleMapsFlutterPlatform.instance;
+
 /// Controller for a single GoogleMap instance running on the host platform.
 class GoogleMapController {
+  /// The mapId for this controller
+  final int mapId;
```

Comment:
Is the mapId a necessary change for the purpose of this PR? Or it is an improvement that we can separate out to a different PR?

Reply:
The mapId is required. Since the platform interface is a stateless singleton, but each map has an instance of a controller, I need to pass something identifying what Map is being controlled to the platform from the controller.